static/app/views/discover/chartFooter.tsx

```diff
@@ -62,6 +62,7 @@ export default function ChartFooter({
       <InlineContainer>{elements}</InlineContainer>
       <InlineContainer>
         <IntervalSelector
+          key={eventView.interval}
```

Comment:
this `key` forces the `IntervalSelector` to re-set it’s internal state once a value has been selected so that we get the initial `items` again.

Reply:
done: 61306b68